.github/workflows/integration.yml

```diff
@@ -137,8 +132,8 @@ jobs:
           composer_name: ${{ inputs.composer_name }}
           magento_version: '2.4.5-p7'
           composer_version: '2'
-  integration-tests-2-4-6-p4-8-1:
-    name: Magento 2.4.6-p4 PHP 8.1 Integration Tests
+  integration-tests-2-4-5-p8-7-4:
+    name: Magento 2.4.5-p8 PHP 7.4 Integration Tests
```

Comment:
Did you test that this works? I would pretty much expect the same issues as in https://github.com/customgento/m2-github-actions/pull/6/files#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bR104.

Reply:
You are right. It is not working and changed accordingly.